bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/transaction/actor/ImportActorMapping.java

```diff
@@ -153,18 +167,15 @@ private void checkAlreadyExistingMembershipMapping(final long actorId, final lon
         } while (actorMembersOfMembership.size() > 0);
     }
 
-    private ActorMapping getActorMappingFromXML() throws SBonitaException {
-        StringReader reader = new StringReader(xmlContent);
+    ActorMapping getActorMappingFromXML() throws SBonitaException {
```

Comment:
no scope ?

IMHO this should not be here and we should have only one contructor. But... you  know ...